sqlmesh/dbt/builtin.py

```diff
@@ -152,11 +152,9 @@ def var(name: str, default: t.Optional[str] = None) -> str:
 
 
 def generate_ref(refs: t.Dict[str, t.Any]) -> t.Callable:
-
-    # TODO suport package name
     def ref(package: str, name: t.Optional[str] = None) -> t.Optional[BaseRelation]:
-        name = name or package
-        relation_info = refs.get(name)
+        ref_name = f"{package}.{name}" if name else package
```

Comment:
Can there be a freak case when the `package` is the same as the `name` in a different package?

Reply:
Never mind, it shouldn't matter.